src/spikeinterface/core/__init__.py

```diff
@@ -146,6 +146,8 @@
 # SortingAnalyzer and AnalyzerExtension
 from .sortinganalyzer import SortingAnalyzer, AnalyzerExtension, create_sorting_analyzer, load_sorting_analyzer
 from .analyzer_extension_core import (
+    SelectRandomSpikes,
```

Comment:
The SelectRandomSpikes needs to be imported since it is now just a normal extension.

Reply:
yes right.